copilot/environments/addons/assessment-import-queue.yml

```diff
@@ -0,0 +1,50 @@
+Parameters:
```

Comment:
Can we switch to `copilot svc deploy` and move the environment to a single repo?

Reply:
I'm pretty sure that a service deploy expects the environment folder to be present